src/go/rpk/pkg/cli/group/describe.go

```diff
@@ -23,14 +25,24 @@ import (
 )
 
 func NewDescribeCommand(fs afero.Fs, p *config.Params) *cobra.Command {
-	var summary, commitsOnly, lagPerTopic bool
+	var summary, commitsOnly, lagPerTopic, re bool
 	cmd := &cobra.Command{
 		Use:   "describe [GROUPS...]",
 		Short: "Describe group offset status & lag",
 		Long: `Describe group offset status & lag.
 
 This command describes group members, calculates their lag, and prints detailed
 information about the members.
+
+The --regex flag (-r) parses arguments as regular expressions
+and describes groups that match any of the expressions.
+
+For example,
+
+    describe foo bar            # describe groups foo and bar
+    describe -r '^f.*' '.*r$'   # describe any group starting with f and any groups ending in r
+    describe -r '*'             # describe all groups
+    describe -r .               # describe any one-character groups
```

Comment:
I know this is similar to what we do in topic delete, but, can we move these examples to the `cobra.Command`'s `Example` field, we are starting to move examples to this field in new commands.

For example: https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cluster/quotas/describe.go#L61-L69

This makes it easier for the doc reference page (example: https://docs.redpanda.com/current/reference/rpk/rpk-security/rpk-security-role-assign/#examples)

Reply:
Addressed.